code/game/machinery/cryopod.dm

```diff
@@ -325,6 +325,12 @@
 		// them win or lose based on cryo is silly so we remove the objective.
 		if(istype(O,/datum/objective/mutiny) && O.target == occupant.mind)
 			qdel(O)
+		else if(is_sacrifice_target(occupant.mind))
+			var/datum/game_mode/cult/cult_mode = ticker.mode
+			O.owner.objectives -= O
+			qdel(O)
+			to_chat(O.owner.current, "<span class='cultitalic'>Our gods chosen gift has gone passed our reach, we have a new goal..Objectives Updated!</span>")
```

Comment:
> god's

> gone past

> new goal... Objectives Updated!

You `qdel` the objective *before* accessing stuff on it. Please don't do that.

This tries to send a message to the *single owner* of the cult objective. Aren't objectives cult-wide? Do they even have an owner set?